src/sasquatchbackpack/cli.py

```diff
@@ -180,10 +192,74 @@ def usgs_earthquake_data(
             click.echo(result)
         click.echo("------")
     else:
-        click.secho("SUCCESS", fg="orange")
+        click.secho("SUCCESS! (kinda)", fg="yellow")
         click.echo("------")
         click.echo("No results found for the provided criteria :(")
         click.echo("------")
+        return
+
+    if not test:
+        click.echo("Sending data...")
+
+        # sasquatch_rest_proxy_url = (
+        #     "https://data-int.lsst.cloud/sasquatch-rest-proxy"
+        # )
+
+        namespace = "lsst.example"
+        topic_name = "usgs-earthquake-data"
+
+        value_schema = json.dumps(
```

Comment:
See the the previous comment on templating your schema. 
Also, it's better to start this code outside cli.py; you can create a new module, for example, `sasquatch.py`, that defines a class with methods to create your topic in Kafka, build the payload, and send the data to Sasquatch.